src/components/engagement_edit_modals/user_edit_fields.tsx

```diff
@@ -41,6 +43,8 @@ export const UserEditFields = ({
   deletedUsers,
   toggleDeleted,
   addUser,
+  setHasValidInput,
```

Comment:
this name is a touch nonconventional, since `set` is typically used for hook state. Maybe `onValidate`?

Reply:
This is actually a hook state. being used for disabling buttons both inside the modal and inside the child of the modal.